lib/render.test.js

```diff
@@ -2,42 +2,34 @@
 
 const _ = require('lodash'),
   filename = __filename.split('/').pop().split('.').shift(),
+  setup = require('./setup'),
```

Comment:
I had to move this require statement from below the `lib` require to above it, otherwise the ordering of the requires made it so that the render library which is also required by setup would be empty